### ISSUE TITLE
Issue with Gauntlet Items

### ISSUE THREAD

**Issue by Brifrost** [equirs/fashionscape-plugin#38]
When using the Gauntlet armour (like LEGS:23894 (Crystal legs (perfected))), There are issues with clipping due to the legs extending down into the boot slot. Also, when wearing capes/backpacks with the (HEAD:23888 (Crystal helm (perfected))), the neck usually disappears on the character. I suspect this is because you do not wear capes at the gauntlet, however. The legs should be fixable though.

I hope this gets resolved soon!

Brifrost

**Comment by equirs** [comment]
I have a feeling these are issues with the models themselves, since they weren't intended to be worn with items outside of the gauntlet. The plugin doesn't do any sort of model manipulation, that would involve a lot more than what the RuneLite API can do.

**Comment by Brifrost** [comment]
> I have a feeling these are issues with the models themselves, since they weren't intended to be worn with items outside of the gauntlet. The plugin doesn't do any sort of model manipulation, that would involve a lot more than what the RuneLite API can do.

Is it possible to hide the boot slot then?  Would be nice to figure out the workaround. I did put on Elven boots and they look better so far, but would love to just go nothing on (if possible with the plugin).

**Comment by equirs** [comment]
Yeah, if you select a boot style in the "base" tab, it'll override any item equipped there. Basically the same effect as hiding the item.

**Comment by Brifrost** [comment]
Yeah just selecting a base feet look still makes the clipping weird.  Just wanted to point it out to you if you didn't know :)

**Comment by equirs** [comment]
Oh, you're right. It looks like this is one of the only items in the game that does that. I'll look into hiding boots when those are equipped.

**Comment by Brifrost** [comment]
Let me know what you find out!
